project1/server/server_tasks/nport_task.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @File Task for monitoring port for 'Normal Commands'
+ */
+
+#ifndef PROJECT1_NPORT_TASK_H
+#define PROJECT1_NPORT_TASK_H
+
+#include "server_task.h"
+#include "../../thread_pool/task.h"
+#include "../../thread_pool/thread_pool.h"
+#include "command_ids.h"
+#include "../file_handler/file_handler.h"
+#include "../file_handler/file_access_manager.h"
+
+namespace server_tasks {
+
+    class NPortTask : public ServerTask {
+    public:
+        thread_pool::Task::Status RunAtomic() final override;
```

Comment:
Don't use both `final` and `override`. Use `final` by default, and `override` if you need to.